app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,128 @@
+module Types
+  class QueryType < Types::BaseObject
+    field :comments, Types::CommentType.connection_type, "Returns all comments", null: false
+    field :comment, Types::CommentType, "Returns comment for ID", null: false do
+      argument :id, ID, required: true, default_value: false
```

Comment:
Do you know whether the `required: true, default_value: false` are the default options? If they are, maybe we can remove them :thinking:.

Reply:
Same here; it doesn't work :relieved:. Feel free to ignore this comment too.